PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -1315,6 +1315,33 @@ public string GetTrueFileName(string file)
             }
         }
 
+        public void LoadProfiles()
+        {
+            ServiceCommon.Log("Loading PowerShell Profiles");
+            PSObject profiles = _runspace.SessionStateProxy.PSVariable.Get("profile").Value as PSObject;
+
+            string sourceProfilesCommand = string.Empty;
+
+            // if a file exists at each location of the profiles, add sourcing that profile to the command
+            foreach (string profileName in DebugEngineConstants.PowerShellProfiles)
+            {
+                PSMemberInfo profileMember = profiles.Members[profileName];
+                var profilePath = (string)profileMember.Value;
+                var profileFile = new FileInfo(profilePath);
+
+                if (profileFile.Exists)
+                {
+                    ServiceCommon.Log(string.Format("Profile file for {0} found at {1}.", profileName, profilePath));
+                    sourceProfilesCommand += string.Format(". '{0}';{1}", profilePath, Environment.NewLine);

```

Comment:
Just declare sorceProfilesCommand and make it equals to the string, which is more explicit and also makes more sense to use IsNullOrEmpty then.